docs/api/saved-objects/import.asciidoc

```diff
@@ -11,11 +11,13 @@ Saved objects can only be imported into the same version, a newer minor on the s
 
 |=======
 | Exporting version | Importing version | Compatible?
-| 6.7.0 | 6.8.1 | Yes
-| 6.8.1 | 7.3.0 | Yes
-| 7.3.0 | 7.11.1 | Yes
-| 7.11.1 | 7.6.0 | No
-| 6.8.1 | 8.0.0 | No
+| 6.7.x | 6.8.x | Yes
+| 6.x.x | 7.x.x | Yes
+| 7.x.x | 8.x.x | Yes
+| 7.1.0| 7.15.x | Yes
```

Comment:
Missing a space here (not sure if this would affect the formatting, but might be worth adding it back to be on the safe side):
```suggestion
| 7.1.0 | 7.15.x | Yes
```

Also this is the only example that lists a specific patch version as you have changed all of the others. Did you mean to use `7.1.x`? IMO it doesn't matter much either way -- the inconsistency just caught my attention.

```suggestion
| 7.1.x | 7.15.x | Yes
```